applications/openshift/general/oauth_logout_url_set/tests/ocp4/e2e-remediation.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+oc patch console.config.openshift.io cluster --type merge -p '{"spec":{"authentication":{"logoutRedirect":"https://my.idp.com/logout"}}}'
```

Comment:
Could you include an empty line at the end, please?

Reply:
done, thank you